src/CSI.IBTA.Customer/Clients/ClaimsClient.cs

```diff
@@ -0,0 +1,30 @@
+﻿using CSI.IBTA.Customer.Endpoints;
+using CSI.IBTA.Customer.Interfaces;
+using CSI.IBTA.Shared.DTOs;
+using Newtonsoft.Json;
+using System.Text;
+
+namespace CSI.IBTA.Customer.Clients
+{
+    internal class ClaimsClient : IClaimsClient
+    {
+        private readonly AuthorizedHttpClient _httpClient;
+
+        public ClaimsClient(AuthorizedHttpClient httpClient)
+        {
+            _httpClient = httpClient;
+            _httpClient.SetBaseAddress("BenefitsServiceApiUrl");
+        }
+
+        public async Task<GenericResponse<PagedClaimsResponse>> GetClaimsByEmployee(int page, int pageSize, int employeeId)
+        {
+
+            var requestUrl = string.Format(BenefitsServiceEndpoints.ClaimsByEmployee, page, pageSize, employeeId);
+            var response = await _httpClient.GetAsync(requestUrl);
+            var responseContent = await response.Content.ReadAsStringAsync();
```

Comment:
could be simplified with https://learn.microsoft.com/en-us/dotnet/api/system.net.http.json.httpcontentjsonextensions.readfromjsonasync?view=net-8.0, to deserialize PagedClaimsResponse automatically